project/src/views/common/common.js

```diff
@@ -3,10 +3,10 @@ const NAV = function (options = {}) {
     <header>
         <img src="assets/images/logo-v2.png" alt="Softvision Logo" class="eval-sv-logo">
         <nav class="eval-header-nav">
-        <a href="" class="eval-header-links eval-header-links-skin default-font">Evaluations</a>
```

Comment:
I would keep the href attribute and give it hash values... this way the location bar will represent the current state of the application

for example
`<a href="#evaluations .... >Evaluations</a>"`

Reply:
👍